### PR TITLE
Screen/home

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Fixed home screen layout

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -5,8 +5,11 @@
   transform: scale(0.9);
   object-fit: contain;
 }
-
+header {
+  /* backdrop-filter: blur(10px);
+  box-shadow: 0 0 1rem var(--primary-color-6); */
+}
```
Comment (bhushannaikaroskar) [inline css/style.css]: remove comments and I guess the whole tag css its empty

**Review by bhushannaikaroskar — APPROVED** [review]
Good  to merge now

File: login.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    <main class="container h-full t-5 mt-5 l-0 r-0 relative grid place-content-center text-dark w-full">
+        <form class="p-5 w-full  bg-light br-sm">
+            <label class="text-2xl mb-2 block text-center text-dark text-400">Login</label>
+            <div class="col flex-gap-1 mb-2">
+                <div class="input-group">
+                    <!-- <label for="username">Username</label> -->
+                    <input type="text" id="username" placeholder="Enter username/email" class="input-dark" required />
+                </div>
+                <div class="input-group">
+                    <!-- <label for="password">Password</label> -->
```
Comment (tonymike99) [inline login.html:59]: Can remove these comments

**Review by tonymike99 — APPROVED** [review]
Nice code. Good to go.

File: login.html (hunk 2 above)
Comment (sanjitsarkar) [reply login.html:59]: Okay I will remove those comments

**Pull request merged**